test/unit/spec/EnrollTotpController_spec.js

```diff
@@ -257,11 +258,10 @@ function (Okta, OktaAuth, LoginUtil, Util, DeviceTypeForm, BarcodeForm,
 
         // Start the enrollment
         test.form.selectDeviceType('APPLE');
+        // Submit would trigger enroll activate call and 1 poll.
         test.form.submit();
 
-        return tick(test)    // 1: submit enrollFactor
-          .then(function () { return tick(test); }) // 2: submit enrollFactor poll
-          .then(function () { return tick(test); }); // Final response tick
+        return tick(test); // To trigger next poll.
```

Comment:
Refactored this to be accurate now. The activate and first poll happens without any delay.